src/client/components/Footer.tsx

```diff
@@ -1,87 +1,178 @@
-  }
+const paddingOverrides = css`
+  /* TODO: Refactor these once https://github.com/guardian/source/pull/820 is merged and published; */
+  padding-left: 0 !important;
+  padding-right: 0 !important;
 `;
 
-const p = css`
-  color: ${palette.neutral[100]};
-  margin: 0;
-  ${textSans.small()};
+const containerStyles = css`
+  display: flex;
+  position: relative;
   ${from.tablet} {
-    width: 320px;
+    border-left: 1px solid ${brand[600]};
+    border-right: 1px solid ${brand[600]};
   }
+  border-bottom: 1px solid ${brand[600]};
 `;
 
-const a = css`
-  color: ${palette.neutral[100]};
+const linkStyles = css`
   text-decoration: none;
-  ${textSans.small()};
+  padding-bottom: ${space[3]}px;
+  display: block;
+  line-height: 19px;
+  color: ${brandText.primary};
+
+  :hover {
+    text-decoration: underline;
+    color: ${brandAlt[400]};
+  }
 `;
 
-const ul = css`
+const ulStyles = css`
   list-style: none;
-  padding: 0;
-  margin: 0;
-  column-count: 2;
-  column-gap: 0;
+`;
+
+const columnStyles = css`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  padding-top: ${space[3]}px;
+  padding-right: 0;
+  padding-bottom: 0;
+  padding-left: ${space[3]}px;
   ${from.tablet} {
-    flex: 1 1 auto;
-    column-count: auto;
-    column-width: 10rem;
-    padding: 0px ${space[6]}px;
+    padding-left: 20px;
   }
+  margin-top: 0;
+  margin-right: ${space[3]}px;
+  margin-bottom: ${space[6]}px;
+  margin-left: 0;
+`;
+
+const leftBorderStyles = css`
+  border-left: 1px solid ${brand[600]};
 `;
 
-const li = css`
-  padding: ${space[1]}px 0;
+const copyrightStyles = css`
+  ${textSans.xxsmall()};
+  /* TODO: Refactor these once https://github.com/guardian/source/pull/820 is merged and published; */
+  padding-left: ${space[3]}px !important;
   ${from.tablet} {
-    padding: 0;
+    padding-left: 20px !important;
+  }
+  padding-right: ${space[3]}px !important;
+  padding-top: ${space[3]}px !important;
+  padding-bottom: ${space[3]}px !important;
+
+  ${until.tablet} {
+    margin-top: ${space[2]}px !important;
   }
 `;
 
-const link = (label: string, href: string) => (
-  <li css={li}>
-    <a css={a} href={href}>
-      {label}
-    </a>
-  </li>
-);
+const backToTopStyles = css`
+  background-color: ${brandBackground.primary};
+  padding: 0 ${space[2]}px;
+  position: absolute;
+  bottom: -21px;
+  right: ${space[3]}px;
+`;
 
 export const Footer = () => (
   <footer css={footer}>
-    <div css={container}>
-      <p css={p}>
-        &copy; {new Date().getFullYear()} Guardian News &amp; Media Limited or
-        its affiliated companies. All rights reserved.
-      </p>
-      <ul css={ul}>
-        {link('Help', locations.HELP)}
-        {link('Terms & Conditions', locations.TERMS)}
-        {link('Contact us', locations.CONTACT_US)}
-        {link('Privacy policy', locations.PRIVACY)}
-        {link('Report technical issue', locations.REPORT_ISSUE)}
-        {link('Cookie Policy', locations.COOKIE_POLICY)}
```

Comment:
Do we not need these two links anymore?

Reply:
> Do we not need these two links anymore?

They're not in the designs so I assume not?
<img width="927" alt="Screenshot 2021-05-25 at 15 41 41" src="https://user-images.githubusercontent.com/1336821/119517709-c2135b00-bd6f-11eb-9530-6bd4f97d7df9.png">